components/Nav/NavPrimaryMobile.tsx

```diff
@@ -1,25 +1,34 @@
+import { FocusEvent, useCallback } from "react"
 import { NavItem } from "./NavItem"
 import { pages } from "../../data/pages"
 import styles from "./NavPrimary.module.css"
 
 export interface INavProps {
 	activeNavLink: string
-	handleNavClick: () => void
+	closeNavMobile: () => void
 }
 
 export const NavPrimaryMobile = ({
 	activeNavLink,
-	handleNavClick,
+	closeNavMobile,
 }: INavProps) => {
+	const handleNavBlur = useCallback((event: FocusEvent<HTMLAnchorElement>) => {
+		const isLastNavItem = /true/i.test(event.target.dataset?.lastNavItem ?? "")
```

Comment:
I'm not really sure what's going on in this line. I've not seen this syntax before. Can you explain what's happening?

Reply:
it's a regex shorthand checking if the tested string contains `true` and returning a boolean. the tested string is either whatever `event.target.dataset?.lastNavItem` returns or an empty string. a needlessly complex way of doing that.